tests/chainerx_tests/unit_tests/routines_tests/test_normalization.py

```diff
@@ -43,10 +43,9 @@ def _create_batch_norm_ndarray_args(
     return x, gamma, beta, mean, var
 
 
-# Note that CUDA (cuDNN) only supports batch normalization with 4 or
-# 5-dimenisional data.
```

Comment:
Could you keep the note until higher dimensional data are supported by `chainerx.batch_norm`?

Reply:
That makes sense! Recovered the comment with an additional note.